packs/blood-lords-bestiary/ghoul-antipaladin.json

```diff
@@ -39,14 +39,9 @@
             "type": "spellcastingEntry"
         },
         {
-            "_id": "kB1mIppy8e01CEe2",
-            "flags": {
-                "core": {
-                    "sourceId": "Compendium.pf2e.spells-srd.Item.jFmWSIpJGGebim6y"
-                }
-            },
+            "_id": "bEdDxGUh3lcv55Pd",
```

Comment:
All the sourceIds were stripped. How are you refreshing?

Reply:
Just dragged the spell again. The refresh button wasn't working for me, which I was going to report tomorrow after I made sure it wasn't already known, or just user error.

Reply:
Yeah, I found the same with the refresh button. https://github.com/foundryvtt/pf2e/issues/16512

If dragging fresh copies is losing the sourceId that may be a separate issue.